source.txt

```diff
@@ -246,11 +246,10 @@ Table of Contents
          and further ancestor folders, using a strong validator [HTTP,
          section 7.2].
 
-    If not exactly one Content-Type header was received as part of a
-    PUT request, the Content-Type header value contains non-ASCII
-    characters, or it is unreasonably long, the server MAY refuse to
-    process the request, and instead respond with a descriptive error
-    message in the body, and a http response code from the 4xx range.
+    If no valid Content-Type header was received as part of a PUT
+    request, the server MAY refuse to process the request, and instead
+    respond with a descriptive error message in the body, and a http
+    response code from the 4xx range.
```

Comment:
1. Just a proposal to improve the form: "and instead respond with a descriptive error message in the body, as well as an HTTP response code from the 4xx range."
2. It's not entirely clear who or what defines "valid" here.